var/spack/repos/builtin/packages/qt-base/package.py

```diff
@@ -89,6 +89,8 @@ class QtBase(QtPackage):
     url = QtPackage.get_url(__qualname__)
     list_url = QtPackage.get_list_url(__qualname__)
 
+    version("6.5.1", sha256="fdde60cdc5c899ab7165f1c3f7b93bc727c2484c348f367d155604f5d901bfb6")
```

Comment:
I found that QtCore did not build properly when using py-pyqt6@6.5 and qt-base@6.4. It's unclear if an exact version match is required or not. I couldn't find much documentation for this.

Reply:
FYI, see also https://github.com/spack/spack/pull/36705/ which also adds the newer version for other qt6 packages.

Reply:
Ah, let me review that